client/src/app/create/create.component.ts

```diff
@@ -0,0 +1,55 @@
+import {Component, OnInit} from '@angular/core';
+import {EventEmitter} from "@angular/common/src/facade/async";
```

Comment:
Widziałem, że niektórzy importują EventEmmiter z paczki '@angular/core'. Ty importujesz jakiś inny? Jeżeli nie to możesz sprawdzić czy z angular/core działa